content/coronavirus_education_page.yml

```diff
@@ -4,12 +4,20 @@ content:
   page_title: "Education and childcare"
   header_section:
     pretext: Guidance for teachers, school leaders, carers, parents and students
+    education-pretext:
+      text: Find out when schools are expected to reopen in
+      scotland: " Scotland, "
+      scotland-url: https://www.gov.scot/news/schools-to-re-open-in-august/
+      wales: "Wales"
+      wales-url: https://gov.wales/how-schools-will-work-during-coronavirus-pandemic#section-38402
+      and: " and "
```

Comment:
there a way to do this in ruby - https://apidock.com/rails/Array/to_sentence

I think that would work better

Reply:
```ruby
[].to_sentence                      # => ""
['one'].to_sentence                 # => "one"
['one', 'two'].to_sentence          # => "one and two"
['one', 'two', 'three'].to_sentence # => "one, two, and three"
```

Reply:
thanks, I'll do that in collections and remove from here later.